content/blog/2020-outreach-partners.md

```diff
@@ -0,0 +1,31 @@
+Title: Bringing PyCon India to a wider audience with your help
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.
+
+<!-- PELICAN_END_SUMMARY -->
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.
+
+Let's make make PyCon India a great conference using the combined effort of our communities. 
+
+## What we expect from you 
+
+We would love for you to spread a word about the conference among your members who are looking to learn something regarding Python. We expect our outreach partners to help us in the following ways: 
+
+* Send emails to your mailing lists and post content on their social media channels on our behalf to make your community members aware of the conference. 

```

Comment:
Both 2nd and 3rd person being used here. Pick one either go with `your` or `their`

Rather emails or being specific, convert it to "Share content on mailing list, social media channels, and messaging groups with your community members about the conference and all its updates,"